### ISSUE TITLE
Incorrect number of arguments in rangerCpp?

### ISSUE THREAD

**Issue by nilescbn** [imbs-hl/ranger#277]
I installed version 0.9.4 (*I originally and mistakenly wrote 0.94*) to try out the class.weights feature. I get this error trying to run my model: 

```
Error in .Call(`_ranger_rangerCpp`, treetype, dependent_variable_name,  : 
  Incorrect number of arguments (39), expecting 38 for '_ranger_rangerCpp'
```
Reverting to the CRAN version (0.9.0), the model runs fine. I've installed and uninstalled twice now to confirm that the error is happening.  

I'm not a sophisticated user so unfortunately I can't help much diagnosing what's going on. Looking at, the [rangerCpp function](https://github.com/imbs-hl/ranger/blob/fae13693167a205d503f7315d43ee8f140682a5c/R/RcppExports.R) I do count 39 arguments. Why it expects 38, I have no idea... 

If more detail would be helpful, I would be happy to provide it. 

Thank you for making this package available. I can't believe how much faster it is than other random forest packages.

**Comment by mnwright** [comment]
That still seems to be an installation problem. Could you try installing again in a fresh R session? Which OS are you using?

**Comment by nilescbn** [comment]
Okay, yes, I was able to run `ranger()` using version 0.9.4 today after re-installing. 

My apolgies, I tried to rule out the installation issue before filing this. Thinking back, although I restarted the R session and tried installing it twice before, I don't think I ever closed down RStudio completely. 

As for the operating system, I'm on Windows 10 using R version 3.4.3. I ran the `devtools::install_github("imbs-hl/ranger")` command from RStudio before. Today, not that it would have made a difference, I ran it from the standard R GUI console. 

Thank you for the reply. I look forward to trying out the new class.weights feature.
